### DISCUSSION TITLE
Plot to follow a neuron accross days

### DISCUSSION THREAD

**Discussion by Electrophysionico** [EnnyvanBeest/UnitMatch#91] [General]
While working with UnitMatch data on Matlab I tried to plot the waveform of a single unit through several days. I wanted to have a subplot for each recording day depicting the average waveform of a unit that has been matched for each recording day. However due to how MatchTable and UniqueIDConversion are structured I find it hard to write a code that only selects a Goodunit that has been recorded and matched through all the recordings and then plot the waveforms. If anyone has an idea or recommendations on how to do that I would be very grateful.

**Comment by cbimbo** [comment]
As discussed in the other discussion, each row of the MatchTable gives you the kilosort ID of each neuron involved in the pair, so you should be able to go back to their waveforms quite easily. Let us know if that didn't help?

**Reply by Electrophysionico** [reply]
Yes sorry, I figured after the discussion, thank you for the help!
